### ISSUE TITLE
Upstream?

### ISSUE THREAD

**Issue by sargun** [lunatik-ng/lunatik-ng#1]
Has this work ever been attempted to be upstreamed?

**Comment by dbausch** [comment]
You mean to Linux? This is way too early. I have a lot of outstanding local changes that will finally make up a major redesign (especially to support multiple/temporary Lua states). Apart from that, I think this change set is quite large (as it includes the whole Lua interpreter into the kernel) and therefore would need strong support from some important people. I do not see why they should at the moment. What is your application for this?

**Comment by sargun** [comment]
Yeah, to Linux. I'm mostly interested in using Lua as a scripting language for some extensions I'm interested in adding to assist in container isolation.

**Comment by uudiin** [comment]
Is there any progress on this now? Thanks.
